features/finders.feature

```diff
@@ -288,12 +288,19 @@ Feature: Filtering documents
     And the page has results region
 
   @javascript
-  Scenario: Facets should be hidden by default on finders except all content
+  Scenario: Facets should not be hidden by default on finders except all content
     When I view the research and statistics finder
     And I should not see a "Show more search options" link
     Then I view the all content finder
     And I should see a "Show more search options" link
 
+  @javascript
+  Scenario: Facets should expand when clicking on the "Show more search options" link on all content
+    When I view the all content finder
+    And I should see a "Show more search options" link
```

Comment:
This and the other test calls like this are currently failing as 'undefined'.

Reply:
Shouldn't be. Let's see if the new build passes